molecule/plain-rhel/molecule.yml

```diff
@@ -99,6 +99,9 @@ provisioner:
   inventory:
     group_vars:
       all:
+        confluent_common_repository_redhat_clients_baseurl: https://packages.confluent.io/clients/rpm/centos/8/x86_64
```

Comment:
I understand these are temporary changes for testing. Lets not keep it in PR. You can override these variables at runtime.